diff --git a/apps/riscv-tests/benchmarks/common/test.ld b/apps/riscv-tests/benchmarks/common/test.ld
index a50b017..d93d5b6 100644
--- a/apps/riscv-tests/benchmarks/common/test.ld
+++ b/apps/riscv-tests/benchmarks/common/test.ld
@@ -29,10 +29,12 @@ SECTIONS
   .tohost : { *(.tohost) }
 
   . = ALIGN(0x1000);
-  .text : { *(.text) }
+  .text : { *(.text .text.*) }
+
+  .rodata : { *(.rodata .rodata.*) }
 
   /* data segment */
-  .data : { *(.data) }
+  .data : { *(.data .data.*) }
 
   .sdata : {
     __global_pointer$ = . + 0x800;
@@ -45,18 +47,18 @@ SECTIONS
     *(.sbss .sbss.* .gnu.linkonce.sb.*)
     *(.scommon)
   }
-  .bss : { *(.bss) }
+  .bss : { *(.bss .bss.*) }
 
   /* thread-local data segment */
   .tdata :
   {
     _tdata_begin = .;
-    *(.tdata)
+    *(.tdata .tdata.*)
     _tdata_end = .;
   }
   .tbss :
   {
-    *(.tbss)
+    *(.tbss .tbss.*)
     _tbss_end = .;
   }
 
