diff --git a/hardware/Makefile b/hardware/Makefile
index 5b8ad66..93759a3 100644
--- a/hardware/Makefile
+++ b/hardware/Makefile
@@ -182,7 +182,7 @@ tests := $(ara_tests) $(cva6_tests)
 .PHONY: verilate
 verilate: $(buildpath) bender $(veril_library)/V$(veril_top)
 
-$(veril_library)/V$(veril_top): $(config_file) Makefile ../Bender.yml $(shell find src -type f) $(shell find ../config -type f) $(shell find include -type f) $(shell find tb -type f) $(shell find deps -type f)
+$(veril_library)/V$(veril_top): $(config_file) Makefile ../Bender.yml $(shell find src -type f) $(shell find ../config -type f) $(shell find include -type f) $(shell find tb -type f) $(shell find deps -type f) $(shell find tb -type f)
 	rm -rf $(veril_library); mkdir -p $(veril_library)
 	$(BENDER) script flist-plus $(bender_targs_veril) $(bender_defs_veril) > $(veril_library)/bender_script_$(config)
 # Verilate the design
